packages/api/src/routes/medical/patient.ts

```diff
@@ -214,6 +212,28 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /patient/:id/consolidated/query/:requestId
+ *
+ * Returns the status and information on a specific consolidated query for a given patient.
+ *
+ * @param req.param.id The ID of the patient whose consolidated query status is to be returned.
+ * @param req.param.requestId The ID of the query status to be returned.
+ * @returns the status and information on a specific consolidated query for a given patient.
+ */
+router.get(
+  "/consolidated/query/:requestId",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const { patient } = getPatientInfoOrFail(req);
+    const requestId = getFrom("params").orFail("requestId", req);
+    const query = patient.data.consolidatedQueries?.find((q: ConsolidatedQuery) => q.requestId === requestId);
```

Comment:
I think this is missing formatting/prettier (likely need to `husky install` on your repo, it should've picked this up).